pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -82,29 +82,28 @@ in
   shortVersion = lib.concatStringsSep "." (lib.take 1 (lib.splitString "." release_version));
 
   pname = "llvm";
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  inherit pname version;
 
   # TODO: simplify versionAtLeast condition for cmake and third-party via rebuild
-  src' = if monorepoSrc != null then
-    runCommand "${pname}-src-${version}" { inherit (monorepoSrc) passthru; } (''
+  src = if monorepoSrc != null then
+    runCommand "llvm-src-${version}" { inherit (monorepoSrc) passthru; } (''
```

Comment:
Think this change is in the wrong commit?  It's in the "move src" one, not the pname one.

Reply:
I though it had to go to the src commit because it was related to the src attribute.

Reply:
Ok, redone in the pname src